apis/v1alpha1/generator.yaml

```diff
@@ -63,9 +67,12 @@ resources:
           operation: ChangeResourceRecordSets
           path: ChangeBatch.Changes.ResourceRecordSet.GeoLocation
       HealthCheckId:
-        from:
-          operation: ChangeResourceRecordSets
-          path: ChangeBatch.Changes.ResourceRecordSet.HealthCheckId
+        # from:
```

Comment:
I don't know if it is ok to change this here without having breaking changes on current deployments.